internal/controller/postgrescluster/pki_test.go

```diff
@@ -365,14 +369,14 @@ func TestReconcileCerts(t *testing.T) {
 		assert.NilError(t, err)
 
 		t.Run("check standard secret projection", func(t *testing.T) {
-			secretCertProj, err := r.reconcileClusterCertificate(ctx, initialRoot, cluster1)
+			secretCertProj, err := r.reconcileClusterCertificate(ctx, initialRoot, cluster1, primaryService)
```

Comment:
:thinking: Looks like our tests currently validate the projection - do we have anything to validate the contents of the certs themselves?  Was looking for something along those lines to verify the DNS names, but looks like our tests are with the projection only.

Should we be doing further validation of the certs themselves here? I do see some other tests in here that appear to grab the cert contents for some additional validation.

Reply:
Added some checks around L440 in this file.